test/benchmark/simulation.cpp

```diff
@@ -61,7 +62,7 @@ TEST_CASE("Benchmark simulators", "[benchmark]")
 
     BENCHMARK("QuickExact")
     {
-        const quickexact_params<lattice> sim_params{sidb_simulation_parameters{2, -0.32}};
+        const quickexact_params<cell<lattice>> sim_params{sidb_simulation_parameters{2, -0.32}};
```

Comment:
If this change was necessary, how did it compile before? This looks like something where only one of the solutions would compile.

Reply:
Yes, the reason is that the benchmark code is not compiled for the CI. Should we change that?

Reply:
Good point. If we do we need to exclude its execution in the test run. Otherwise, this would just consume too much time. I'll take a note of this. Later problem 😄

Reply:
Yes, that's the point. Then, we just leave it for now. Agreed! 😁